front/src/modules/simulationResult/components/ChartHelpers/drawHelpers.ts

```diff
@@ -15,4 +19,17 @@ const getAxisValues = (
   middle: segment[`${keyValues[keyIndex]}_middle`],
   end: segment[`${keyValues[keyIndex]}_end`],
 });
+
+/** Returns the start, middle and end values of the heights or the positions
+ * (depending on the axis and rotation) of a given segment. */
+export const getAxisValuesV2 = (
```

Comment:
Remove this function and only adapt the args of getAxisValues
```
const getAxisValues = (
  segment:
    | ElectricalConditionSegment
    | PowerRestrictionSegment
    | ElectricalConditionSegmentV2
    | PowerRestrictionSegmentV2,
```

Reply:
Done. `PowerRestrictionSegmentV2` wasn't used anymore so I removed it, and added a comment to drop `ElectricalConditionSegment` when dropping v1.